model/task/db.go

```diff
@@ -1031,11 +1039,15 @@ func FindByExecutionTasksAndMaxExecution(taskIds []string, execution int, filter
 				"$lte": execution,
 			},
 		}
+		oldTaskPipeline = append(oldTaskPipeline, bson.M{"$match": match})
+		oldTaskPipeline = append(oldTaskPipeline, addDisplayStatus)
+		filterMatch = bson.M{}
```

Comment:
Might be missing context but is there a reason to not reuse the filterMatch from above?

Reply:
No reason at all I just missed it.